arkouda-env-dev.yml

```diff
@@ -0,0 +1,28 @@
+name: arkouda-env-dev
+channels:
+  - conda-forge
+dependencies:
+  - python>=3.7   # minimum 3.7
+  - numpy>=1.16.5,<=1.21.5
+  - pandas>=1.1.0
+  - pyzmq>=20.0.0
+  - tabulate
+  - pyfiglet
+  - versioneer
+  - matplotlib
+  - h5py
+  - pip
+  # Developer dependencies
+  - pexpect
+  - pytest
+  - pytest-env
+  - Sphinx
+  - sphinx-argparse
+  - sphinx-autoapi
+  - typed-ast
+  
+  - pip:
+    - typeguard==2.10.0
+    # Developer dependencies
+    - mypy==0.931

```

Comment:
According to `setup.py` this should be `mypy>=0.931`. I know there were issues when your version was higher. Can we track that down and determine which case is correct and set it? We can probably add a separate issue to track this.

Reply:
agree with this comment.
https://github.com/Bears-R-Us/arkouda/blob/b9e53938bce78a34be0b6f1631525f5b1db95f52/setup.py#L154-L156

Reply:
Updated to by `mypy>=0.931`
I will add an issue to track down potential version issue